workshop/content/030_basic_content/030_basic_experiment/30-experiment-cli/_index.en.md

```diff
@@ -187,11 +187,11 @@ Before using this template, please ensure that you replace the ARN for the FIS e
 
 ## Working with templates
 
-The rest of this section uses the [**AWS CLI**](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html). If you are using [**AWS Cloud9**](https://console.aws.amazon.com/cloud9/home/product) this should work out of the box. Otherwise please ensure you have configured AWS credentials for the CLI.
+The rest of this section uses the [**AWS CLI**](https://aws.amazon.com/cli/). If you are using [**AWS Cloud9**](https://console.aws.amazon.com/cloud9/home/product) this should work out of the box. Otherwise, please ensure you have installed [**AWS CLI**](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) and configured [AWS credentials for the CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config).
```

Comment:
I understand that it's tempting to use the google search result here but that page has a lot of chaff on it. Using the proper [documentation link](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) instead would be better but still makes it hard to find how to install it if needed. Using the install link originally in the document makes it easy to navigate back to the "what is" page but also creates a straightforward entry point for anyone that still needs to install it.

I will fix this before approving